github/ci/prow-deploy/files/jobs/kubevirt/terraform-provider-kubevirt/terraform-provider-kubevirt-presubmits.yaml

```diff
@@ -0,0 +1,76 @@
+presubmits:
+  kubevirt/terraform-provider-kubevirt:
+  - name: pull-terraform-provider-kubevirt-e2e-k8s-1.20
+    skip_branches:
+      - release-\d+\.\d+
+    annotations:
+      fork-per-release: "true"
+      testgrid-create-test-group: "false"
+    always_run: true
+    optional: false
+    skip_report: false
+    cluster: prow-workloads
+    decorate: true
+    decoration_config:
+      timeout: 1h
+      grace_period: 5m
+    max_concurrency: 11
+    labels:
+      preset-dind-enabled: "true"
+      preset-docker-mirror-proxy: "false"
+      preset-shared-images: "true"
+      preset-bazel-cache: "false"
```

Comment:
If your job uses bazel we can enable now `preset-bazel-cache` and `preset-bazel-unnested` in the prow-workloads cluster, it will improve the execution time.

Reply:
In e2e-tests changed to true and removed those lines from unit-tests